Minio/AWSS3Endpoints.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
```

Comment:
Both Core and Framework run under Minio namespace. The issue here is that code in MinioCore2 is a .NetCore class library that compiles for both Core and Framework, however the Core tests run fine while the Framework give AWS signature mismatch ( this is after stripping the port from host name, that was tripping me up earlier).

Maybe the better strategy would be to merge to minio-dotnet after Core and Framework code base is unified. Nitish can work from my fork until then.